src/MAUI/Maui.Samples/Samples/Symbology/StyleGeometryTypesWithSymbols/readme.metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+    "category": "Symbology",
+    "description": "Use a symbol to display a geometry on a map.",
+    "formal_name": "StyleGeometryTypesWithSymbols",
+    "ignore": false,
+    "images": [
+        "stylegeometrytypeswithsymbols.jpg"
+    ],
+    "keywords": [
+        "display",
+        "fill",
+        "graphics",
+        "line",
+        "marker",
+        "overlay",
+        "picture",
+        "point",
+        "symbol",
+        "visualization"
+    ],
+    "offline_data": [],
+    "redirect_from": [
+        "/net/latest/maui/sample-code/style-geometry-types-with-symbols.htm"
```

Comment:
I think we should update this to include the redirects from the removed samples.